src/pkd-query/pkd-query.controller.ts

```diff
@@ -1,4 +1,11 @@
 import { Controller } from '@nestjs/common';
+import { PkdQueryService } from './pkd-query.service';
 
 @Controller('pkd-query')
-export class PkdQueryController {}
+export class PkdQueryController {
+  constructor(private readonly pkdQueryService: PkdQueryService) {}
+
+  getPkd(profession: string) {
```

Comment:
since we're in context of "PkdQueryController" we know by default that "get" is supposed to return pkd. Having it in name feels redundant. (Same goes for service)
```suggestion
  get(profession: string) {
```